src/mapml.js

```diff
@@ -2428,8 +2732,14 @@ M.TemplatedTileLayer = L.TileLayer.extend({
     initialize: function(template, options) {
       // _setUpTileTemplateVars needs options.crs, not available unless we set
       // options first...
+      let inputData = M.extractInputBounds(template);
+      this.zoomBounds = inputData.zoomBounds;
+      this.layerBounds=inputData.bounds;
+      this.isVisible = true;
+      Object.assign(options,this.zoomBounds);
```

Comment:
Instead of using Object.assign there are Leaflet utilties for this, for example on line 2750 we use L.extend(options, {...}) to extend the leaflet options before passing to the initialize call.  We should stick to this style; I think it enhances readability.